include/boost/winapi/process.hpp

```diff
@@ -366,7 +390,15 @@ using ::OpenProcess;
 #if BOOST_WINAPI_PARTITION_DESKTOP_SYSTEM
 
 using ::GetExitCodeProcess;
+#if !defined(_WIN32_WCE)
 using ::ExitProcess;
+#endif
+
+BOOST_FORCEINLINE BOOST_NORETURN boost::winapi::VOID_ exit_process(
+    boost::winapi::UINT_ uExitCode)
+{
+    ExitProcess(uExitCode);
```

Comment:
Where is this macro defined on CE? You haven't added any #includes to this header.

Reply:
Good catch, thanks. To support compilation without including any headers I will call TerminateProcess directly instead of "calling" ExitProcess macro.

Reply:
Yes, though no need for the additional parenthesis around `uExitCode` since we're not in a macro.